.gitattributes

```diff
@@ -0,0 +1,3 @@
+/Tests export-ignore
+/.gitignore export-ignore
+/phpunit.xml.dist export-ignore
```

Comment:
what's about `.php_cs`, `.travis.yml`, `phpstan.neon.dist`? (just asking)

Reply:
I did not think about that but we can add them